lib/liblastlog2.map

```diff
@@ -1,5 +1,6 @@
 LIBLASTLOG2_1.0 {
   global:
+        ll2_check_database
```

Comment:
/usr/bin/ld:/home/runner/work/lastlog2/lastlog2/lib/liblastlog2.map:4: syntax error in VERSION script

Beside the syntax error, as this is a new function, you should use a new symbol version "LIBLASTLOG2_1.2" for this function.

Reply:
Thanks for your feedback. Syntax fixed and symbol version update to LIBLASTLOG2_1.2 as you suggested.